src/main/resources/twitter-demo/newTweetBook.tpl

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8' />
+    <title></title>
+    <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
+    
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.14.0/mapbox-gl.css' rel='stylesheet' />
+    <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet" type="text/css">
+     <link href="static/css/dc.css" rel="stylesheet" type="text/css">
+     <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
+     <link href='https://api.mapbox.com/mapbox.js/v2.3.0/mapbox.css' rel='stylesheet' />
+
+    <script src="static/js/asterix-sdk-stable.js"></script>
+    <script src="static/js/rainbowvis.js"></script>
+    <script src="http://code.jquery.com/jquery.min.js"></script>
+     <script src="static/js/bootstrap.min.js"></script>
+     <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.14.0/mapbox-gl.js'></script>
+     <script src='https://api.mapbox.com/mapbox.js/v2.3.0/mapbox.js'></script>
+     <script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>
+     <script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
+      <script src="static/js/crossfilter.min.js"></script>
```

Comment:
Sorry, it seems that my sublime's indentation is too small, can't tell difference